scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -210,6 +216,19 @@ def discover(self):
                     logger.debug(f'Quarantine file {quarantine_file} is empty')
             self.quarantine = Quarantine(ql)
 
+        #handle extra scripts
```

Comment:
here and in other places 

```suggestion
        # handle extra scripts
```